apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -78,7 +86,7 @@ export class WorkshopForm extends React.Component {
       id: PropTypes.number.isRequired,
       facilitators: PropTypes.array.isRequired,
       location_name: PropTypes.string.isRequired,
-      location_address: PropTypes.string.isRequired,
+      location_address: PropTypes.string,
```

Comment:
Fixes unrelated console error -- oversight in my [previous PR making address optional](https://github.com/code-dot-org/code-dot-org/pull/35031).